packages/create-plugin/templates/app/src/components/App/App.tsx

```diff
@@ -1,8 +1,21 @@
-import * as React from 'react';
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
 import { AppRootProps } from '@grafana/data';
+import { ROUTES } from '../../constants';
+import { PageFour, PageOne, PageThree, PageTwo } from '../../pages';
+import { prefixRoute } from '../../utils/utils.routing';
 
-export class App extends React.PureComponent<AppRootProps> {
-  render() {
-    return <div className="page-container">Hello Grafana!</div>;
-  }
+export function App(props: AppRootProps) {
+  return (
+      <Switch>
+        <Route exact path={prefixRoute(ROUTES.Two)} component={PageTwo} />
+        <Route exact path={prefixRoute(`${ROUTES.Three}/:id?`)} component={PageThree} />
+
+        {/* Full-width page (this page will have no side navigation) */}
+        <Route exact path={prefixRoute(ROUTES.Four)} component={PageFour} />
```

Comment:
I am afraid this is not true in this case (full-width page). In the old navigation it was done [by returning an empty navModel](https://github.com/grafana/grafana-plugin-examples/blob/main/examples/app-basic/src/utils/utils.routing.ts#L24), while in the new navigation it would be done by not rendering a page (PluginPage) component. 

Maybe we can just remove it and update the examples (so they can find it in there)?

Reply:
Silly me, it is using the `canvas` layout. Let me resolve this comment.